lib/src/screens/forgot_password_screen.dart

```diff
@@ -0,0 +1,342 @@
+    double contextPadding = 0;
+    double iconAndTopSpacing = 0;
+    double iconWidth = 0;
+    double iconHeight = 0;
+    double iconAndTextSpacing = 0;
+    double startTextAndIconSpacing = 0;
+    double containerSpacing = 0;
+    double containerWidth = 0;
+    double containerHeight = 0;
+    double topContainerSpacing = 0;
+    double resetPasswordFontSize = 0;
+    double resetPasswordTextLetterSpacing = 0;
+    double tutorialFontSize = 0;
+    double tutorialLetterSpacing = 0;
+    double tuttorialSpacing = 0;
+    double emailTextSpacing = 0;
+    double enterEmailFieldSpacing = 0;
+    double buttonSpacing = 0;
+    double heightButton = 0;
+    double buttonFontSize = 0;
+
+    if (Device.get().isPhone) {
+      crossAxisAlignment = CrossAxisAlignment.start;
+      contextPadding = widthScreen * 0.25 / 3.75;
+      iconAndTopSpacing = heightScreen * 0.6 / 8.12;
+      mainAxisAlignment = MainAxisAlignment.start;
+      iconWidth = widthScreen * 0.18 / 3.75;
+      iconHeight = heightScreen * 0.26 / 8.12;
+      iconAndTextSpacing = widthScreen * 0.1 / 3.75;
+      containerWidth = widthScreen;
+      containerHeight = heightScreen;
+      topContainerSpacing = heightScreen * 0.5 / 8.12;
+      resetPasswordFontSize = 32;
+      resetPasswordTextLetterSpacing = 0.5;
+      tuttorialSpacing = heightScreen * 0.24 / 8.12;
+      tutorialFontSize = 15;
+      tutorialLetterSpacing = 0.5;
+      emailTextSpacing = heightScreen * 1.06 / 8.12;
+      enterEmailFieldSpacing = heightScreen * 0.2 / 8.12;
+      buttonSpacing = heightScreen * 0.53 / 8.12;
+      heightButton = heightScreen * 0.5 / 8.12;
+      buttonFontSize = 24;
+    } else if (Device.get().isTablet) {
+      crossAxisAlignment = CrossAxisAlignment.center;
+      iconAndTopSpacing = heightScreen * 0.8 / 10.24;
+      mainAxisAlignment = MainAxisAlignment.center;
+      iconWidth = widthScreen * 0.18 / 7.68;
+      iconHeight = heightScreen * 0.26 / 10.24;
+      iconAndTextSpacing = widthScreen * 0.1 / 7.68;
+      startTextAndIconSpacing = heightScreen * 1.3 / 10.24;
+      containerSpacing = heightScreen * 0.36 / 10.24;
+      containerWidth = widthScreen * 4.25 / 7.68;
+      containerHeight = heightScreen * 5.47 / 10.24;
+      contextPadding = widthScreen * 0.5 / 7.68;
+      topContainerSpacing = heightScreen * 0.5 / 10.24;
+      tuttorialSpacing = heightScreen * 0.39 / 10.24;
+      resetPasswordFontSize = 40;
+      tutorialFontSize = 16;
+      emailTextSpacing = heightScreen * 0.5742 / 10.24;
+      enterEmailFieldSpacing = heightScreen * 0.2 / 10.24;
+      buttonSpacing = heightScreen * 0.2758 / 10.24;
+      heightButton = heightScreen * 0.5 / 10.24;
+      buttonFontSize = 16;
+    }
+
+    return Scaffold(
+        body: SingleChildScrollView(
+      child: Form(
+        key: _form,
+        child: Stack(
+          children: [
+            if (Device.get().isTablet)
+              (ShaderMask(
```

Comment:
ShaderMask what is this?

Reply:
I watched UI design video on youtube. And the ShaderMask can apply gradients to a background image widget. So I apply ShaderMask to my project.